src/integration.test.ts

```diff
@@ -531,3 +598,45 @@ describe(`Integration test for portal ${portal}`, () => {
     });
   });
 });
+
+/**
+ * Runs the full etag test on the given skylinks.
+ *
+ * @param skylink1 - The first skylink.
+ * @param skylink2 - The second skylink.
+ */
+export async function etagTestSkylinks(skylink1: string, skylink2: string): Promise<void> {
```

Comment:
I'd like to change this function name so it's more clear what it does, non blocking though

```suggestion
export async function expectDifferentEtags(skylink1: string, skylink2: string): Promise<void> {
```